src/pybind/mgr/nfs/module.py

```diff
@@ -57,6 +59,7 @@ def _cmd_nfs_export_create_rgw(
             readonly: Optional[bool] = False,
             client_addr: Optional[List[str]] = None,
             squash: str = 'none',
+            sectype: Optional[List[str]] = None,
```

Comment:
Document it here,
https://github.com/ceph/ceph/blob/main/doc/mgr/nfs.rst#create-rgw-export